server/integration-tests/sqlite_test.hs

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env stack
+-- stack --stack-yaml ../../client-haskell/stack.yaml runghc --package sqlite-simple --package aeson
+
+{-# OPTIONS_GHC -Wall #-}
+
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+{- | Test whether SQLite was installed with JSON support enabled by calling the "json" SQL
```

Comment:
This comment looks misplaced, was it supposed to go elsewhere?

Reply:
I updated the comment now.